src/model/smp.c

```diff
@@ -15,7 +15,15 @@ void migrateTCB(tcb_t *tcb, word_t new_core)
 #ifdef CONFIG_DEBUG_BUILD
     tcbDebugRemove(tcb);
 #endif
-    Arch_migrateTCB(tcb);
+#ifdef CONFIG_HAVE_FPU
+    /* If the thread owns the FPU of the core it is currently running on (which
+     * is not necessarily the core, that we are now running on), then release
+     * this cores's FPU.
```

Comment:
Good point, the migrated TCB doesn't need to be running on the current CPU core. So fix the comment all the way:

"then release that core's FPU."

Reply:
@kent-mcleod was too fast, so I'll fix my typos in another PR then.

Reply:
Well, "this" is ambiguous enough and can refer to the correct core, I thought it would be slightly clearer if "that" was used, but no big deal.

Reply:
oops sorry. I reviewed that the initial feedback had been action-ed but missed the follow up comment